### ISSUE TITLE
train with images that don't have any object in them

### ISSUE THREAD

**Issue by bparaj** [thtrieu/darktf#761]
If I want to train YOLO with images that contain no objects, and thus, no bounding boxes in them, what would be the correct way to feed in the annotations?

For now, I have given an arbitrary class name (say "scratch") and set their bounding box parameters to 0. Training doesn't seem to complain explicitly but it could be affecting the IOU loss computation (division by zero).

**Comment by youyuge34** [comment]
I believe it has no need or effect to add some images that contain no objects into the training set.
Knowning that in one normal image, the bounding boxes which contains no objects have large percentage already (which we called `negative sample`). What we optimize in the paper is to implement some tricks to reduce those `negative samples effect`. For example in paper of YOLOv2, give the loss of negative samples a small parameter. 
So if u still add images  that contain no objects will cause the result worse than better. 
U should reduce the `negative samples` instead of increasing them.
